### PR TITLE
Moved the language dropdown to the settings menu

### PR DESCRIPTION
**Pull request by TomMalbran**
screen shot 2016-09-11 at 10 11 19 pm

### CODE REVIEW

**Comment by TomMalbran** [comment]
I went ahead and made the saving of the language change after you click on save and remove the language toggle since is not required anymore.

**Comment by Tranberry** [comment]
Isn't there ui elements in the 'new ui' that could be used here? Anyway great change imo.

File: Assets/Scripts/Localization/LocalizationTable.cs [hunk 1]
```diff
@@ -64,6 +64,13 @@ public static string GetLocalization(string key, params string[] additionalValue
             return GetLocalization(key, FallbackMode.ReturnDefaultLanguage, currentLanguage, additionalValues);
         }
 
+        public static void SetLocalization(int lang)
+        {
+            string[] languages = GetLanguages();
+            currentLanguage = languages[lang];
+            Settings.SetSetting("localization", languages[lang]);
```
Comment (Grenkin1988) [inline Assets/Scripts/Localization/LocalizationTable.cs]: I would put this in `DialogBoxSettings`

**Comment by koosemose** [comment]
Works Beautifully. Merged.

**Pull request merged**